docs/src/guides/defaults.md

```diff
@@ -45,3 +45,17 @@ npoint(::QuadTrait, geom) = 4
 npoint(::PentagonTrait, geom) = 5
 npoint(::HexagonTrait, geom) = 6
 ```
+
+# Implementations
+GeoInterface is implemented automatically for `NTuple`s, `NamedTuple`s and `Vector`s to behave as Points.
```

Comment:
Perhaps good to be precise; Vector should be AbstractVector, and eltype for all should be a Real.